wolfcrypt/src/pkcs12.c

```diff
@@ -316,23 +316,23 @@ static int GetSafeContent(WC_PKCS12* pkcs12, const byte* input,
      * through the ContentInfo's and add them to our
      * AuthenticatedSafe struct */
     {
-        int CISz;
-        ret = GetSequence(input, &localIdx, &CISz, size);
+        word32 CISz;
+        ret = GetSequence(input, &localIdx, (int *)&CISz, size);
```

Comment:
Can't do this.
int is not always 32 bits.

Reply:
:heavy_check_mark: